daisy/common.go

```diff
@@ -29,6 +29,9 @@ import (
 	"google.golang.org/api/compute/v1"
 )
 
+// FullResourceURLPrefix represents the prefix of full resource url
+var FullResourceURLPrefix = "https://www.googleapis.com/compute/[^/]*/"
```

Comment:
This var doesn't have to be exported since it's only used in daisy package.

Reply:
removed